### ISSUE TITLE
Inconsistent indentation on function body

### ISSUE THREAD

**Issue by chorvus** [clutchski/coffeelint#64]
Hello, trying to set the style for my mysql calls and coffeelinting the code I get an "inconsistent indentation" warning on the callback function body. Indenting in the same level as the arguments and function declaration removes the warning but I think it should not raise any warnings at it.

Is it the intended behavior?

```
mysql.query \
    """
    SELECT
        `user_id` as `id`,
        `user_name` as `name`,
        `user_password` as `password`
    FROM
        `users`
    WHERE
        `user_name` = ?
    LIMIT 1
    """,

    ['test'],

    (err,res) ->
        console.log err
        console.log res[0]
```

Also, thank you for the good work! :)
